bsmetadata/hydra_configs/v2.yaml

```diff
@@ -27,8 +27,8 @@ data_config:
     metadata_sep: ' | '
     metadata_key_value_sep: ': '
     metadata_probability: 1.0
-    treat_local_metadata_as_regular_text: false
-    add_local_metadata_special_tokens_in_prefix: false
+    treat_local_metadata_as_regular_text: true
+    add_local_metadata_special_tokens_in_prefix: true
```

Comment:
I would suggest adding a parameter for the local metadata as well to allow their controllability afterwards.
A prompt beginning with `html |||` would triggers the addition of html tags inside afterwards. If the prompt doesn't begin with `html |||`, then we shouldn't have html tags in the text generated

Reply:
> A prompt beginning with html ||| would triggers the addition of html tags inside afterwards. If the prompt doesn't begin with html |||, then we shouldn't have html tags in the text generated

Sound good to me!

> I would suggest adding a parameter for the local metadata as well to allow their controllability afterwards.

I don't understand this. Does this change already does what you want? Or are you suggesting to add a different parameter?

Reply:
> Does this change already does what you want? 

yes exactly! My comment applies to all local-type metadata (for a long time the entities were also local-type metadata, I don't know what is the status currently). The idea is exactly what I explained in the following sentences: `html` is the parameter for the local metadata "html" :smile: 

Don't hesitate to tell if this is still confusing :smile:

Reply:
> for a long time the entities were also local-type metadata, I don't know what is the status currently

Entities are still local-type metadata. :slightly_smiling_face: